lib/repo_tools.pm

```diff
@@ -267,23 +275,34 @@ C<$datafile> is repository source.
 
 =cut
 sub rmt_import_data {
-    my ($datafile) = @_;
-    my $datapath = "/mnt/external/";
-    # Decompress the RMT data file to test path
-    assert_script_run("mkdir -p $datapath");
-    assert_script_run("wget -q " . data_url("rmt/$datafile"));
-    assert_script_run("tar -xzvf $datafile -C $datapath");
-    assert_script_run("rm -rf $datafile");
+    my ($datapath) = @_;
+    # Check import data resource exsited
+    assert_script_run("ls $datapath");
     # Import RMT data from test path to new RMT server
     assert_script_run("rmt-cli import data $datapath",  600);
     assert_script_run("rmt-cli import repos $datapath", 600);
-    # Show repo list on new RMT server for later debugging
```

Comment:
After import data, we need verify it by show enabled repo from the original RMT server.